packages/ui/src/components/TimelineEntry.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactNode } from 'react'
+import { twMerge } from 'tailwind-merge'
+
+// TODO(JJ):
+// - Add max num periods?
+
+interface Props {
+  children?: ReactNode
+  periodIx: number
+  segmentIx: number
+  numSegments: number
+  gameStatus: string
+  entryStatus: 'PAST' | 'CURRENT' | 'FUTURE'
+  periodName?: string
+  segmentName?: string
+}
+
+function TimelineEntry({
+  children,
+  periodIx,
+  segmentIx,
+  numSegments,
+  gameStatus,
+  entryStatus,
+  periodName = 'Period',
+  segmentName = 'Segment',
+}: Props) {
+  let id = ''
+  let styling = ''
+  switch (entryStatus) {
```

Comment:
this can be done with twMerge for conditional styling (like this it is kind of redundant). see below

Reply:
Very true! I find your suggestion more readable. Though, we evaluate the if statement 3 times, which prop. costs nothing.